Solutions/CohesitySecurity/Tools/grant_keyvault_permission.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/zsh
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+# Description: This script grants "Get" permissions to the specified user and playbooks on the Key Vault.
+
+# Set variables
+keyvault_prefix="cohesitypro"
```

Comment:
I though we already have a config variable for the prefix. Please reuse it.

Reply:
done